ui/js/controllers/jobs.js

```diff
@@ -109,6 +109,48 @@ treeherderApp.controller('ResultSetCtrl', [
             $rootScope.selectedJob = job;
         };
 
+        $rootScope.$on(thEvents.applyNewJobs, function () {
+            $scope.getDecisionTaskVisibility();
+        });
+
+        $scope.getDecisionTaskVisibility = function () {
+            let dtId = ThResultSetStore.getGeckoDecisionTaskId($scope.repoName, $scope.resultset.id);
```

Comment:
I suppose we could also use `getGeckoDecisionJob()`, but `getGeckoDecisionTaskId()` and it's property 'No decision task' seemed more precise to me.